Octokit.Tests.Integration/fixtures/OrganizationsHooksFixture.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Octokit.Tests.Integration.fixtures
+{
+    public class OrganizationsHooksFixture : IDisposable
+    {
+        readonly IGitHubClient _github;
+        readonly OrganizationHook _hook;
+        private string organizationFixture;
+
+        public OrganizationsHooksFixture()
+        {
+            _github = Helper.GetAuthenticatedClient();
+            organizationFixture = Helper.Organization;
+            _hook = CreateHook(_github, organizationFixture);
+        }
+
+
+        public string org { get { return organizationFixture; } }
+
+        public OrganizationHook ExpectedHook { get { return _hook; } }
+
+        public void Dispose()
+        {
+            _github.Organization.Hooks.Delete(organizationFixture,_hook.Id);
+        }
+
+        static OrganizationHook CreateHook(IGitHubClient github, string _organizationFixture)
```

Comment:
The integration tests weren't working, and the error was something related to fixtures, hence I tried out some things maybe then it got changed.